servicecontrol/plugins/index.md

```diff
@@ -32,15 +49,15 @@ NOTE: For NServiceBus version-dependent requirements for each plugin, review the
 
 Once deployed on an active endpoint, the endpoint sends plugin-specific information to ServiceControl. Plugins send messages using the defined endpoint transport to the ServiceControl queue. Location of ServiceControl queue is determined by the following:
 
-1. **Endpoint`s configuration file** 
+1. **Endpoint`s configuration file**
 Check for an `appSetting` named `ServiceControl/Queue` e.g. `<add key="ServiceControl/Queue" value="particular.servicecontrol"/>`.
-1. **Convention based on the configured Error Queue machine** 
+1. **Convention based on the configured Error Queue machine**
```

Comment:
Is it actually possible to not configure an error queue? Would we really expect someone to configure an audit queue but not an error queue?